libs/utils/block-notifications.js

```diff
@@ -89,7 +89,10 @@ export default async function blockNotifications(base) {
     const { body } = document;
     body.classList.add('block-notifications');
 
-    decorateNotificationControls(body);
+    // Stop duplicate show/hide controls
+    if (!document.querySelector('.notification-controls-container')) {
+      decorateNotificationControls(body);
+    }
```

Comment:
This does the job, but I'm wondering whether it brings any value to have it here inside the loop or if it can be moved outside of it and thus avoid the conditional being run multiple times as well.